contrib/automation_tests/test_cases/symbols_tab.py

```diff
@@ -217,13 +217,29 @@ def _check_and_update_duration(self, storage_key: str, current_duration: float,
 
 class WaitForLoadingSymbolsAndCheckModule(E2ETestCase):
```

Comment:
Personal preference: I would like to rename this as `WaitForLoadingSymbolsAndCheckModuleState` 
as now it is not just checking module is loaded.

And `WaitForLoadingSymbolsAndCheckNoErrors` --> `WaitForLoadingSymbolsAndCheckNoErrorStates` .

Reply:
And `VerifyModuleLoaded` as `VerifyModuleState`

Reply:
Sounds reasonable, done. It touches a few files, fingers crossed.